MCprep_addon/optimize_scene.py

```diff
@@ -26,58 +26,77 @@
 MAX_BOUNCES = 8 # 8 is generally the standard for light bounces
```

Comment:
File-level comment:

What are some basic tests we can run and add to the test runner, to make sure this is working across versions? This is probably a decent time to add something even basic, so I can validate it continues to work across versions.

Reply:
Reminder to think of this, I'd like to have at least one unit test so we can have confidence when testing new (and old) blender versions accordingly.